_posts/2018-02-12-release-3.2.1.markdown

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title:  "Release: 3.2.1"
```

Comment:
Do we need the package name in the title, i.e. "Release: OMERO.figure 3.2.1"?

Reply:
This is posted on the Figure blog not the main website so it's not needed